### DISCUSSION TITLE
Push to pypi

### DISCUSSION THREAD

**Discussion by cullzie** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#194] [Ideas]
@aapatre @fakeid30 
Creating this thread to discuss pushing to pypi

The only things which I think are left to do are:
- [x] Add author to `setup.py` 
- [x] Add author_email to `setup.py`
- [x] Add maintainer to `setup.py`
- [x] Update documentation

Please feel free to add to the checklist above ^

**Comment by fakeid30** [comment]
We should create a dedicated email for this project and use that as the email where we all will have access to

**Comment by fakeid30** [comment]
And for authors we can have all 3 of our names .

**Reply by cullzie** [reply]
There is also a `maintainer` field. `author` should probably just be the original repo owner

**Comment by cullzie** [comment]
Here's a test publication I've done on TestPypi if you want to check its ok: https://test.pypi.org/project/udemy-enroller/2.0.0/
Email is just a dummy one at the moment but you can download it and test it out.

**Reply by cullzie** [reply]
I've claimed this email `udemyenroller@gmail.com`

**Reply by fakeid30** [reply]
Good work cullzie. If you can, please cut a release. I'm out for some real life stuff.

**Reply by cullzie** [reply]
Ye I'll leave it until Saturday to make sure there are no more updates to comidoc. And I'll be able dedicate a bit more time to it.

**Comment by cullzie** [comment]
Officially released to PyPi: https://pypi.org/project/udemy-enroller/
